spec/graphql/mutations/repository/remove_url_mapping_mutation_spec.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Mutations::Repository::RemoveUrlMappingMutation do
+  let!(:user) { create :user }
+  let(:url_mapping) { create(:url_mapping) }
+  let(:repository) { url_mapping.repository }
+  before do
+    repository.add_member(user, 'admin')
+  end
+
+  let(:context) { {current_user: current_user} }
+
+  let(:url_mapping_id) { url_mapping.to_param }
+  let(:repository_id) { repository.to_param }
+  let(:variables) do
+    {'urlMappingId' => url_mapping_id,
+     'repositoryId' => repository_id}
+  end
+
+  let(:result) do
+    OntohubBackendSchema.execute(
+      query_string,
+      context: context,
+      variables: variables
+    )
+  end
+
+  let(:query_string) do
+    <<-QUERY
+    mutation ($repositoryId: ID!, $urlMappingId: ID!) {
+      removeUrlMapping(repositoryId: $repositoryId, urlMappingId: $urlMappingId)
+    }
+    QUERY
+  end
+
+  subject { result }
+
+  shared_examples 'handling the deletion' do
+    it 'deletes Url Mapping' do
+      expect(subject['data']['removeUrlMapping']).
+        to eq(expectation_response_matcher)
+    end
+
+    it 'deletes Url Mapping, which already has been deleted' do
+      subject
+      expect(UrlMapping.first(id: url_mapping.id)).
+        to be(expectation_database_object)
```

Comment:
If the deletion was successful, we always expect it to return `nil` afterwards, so you can just hardcode it here instead of passing it via `expectation_database_object`.